query-compiler/query-compiler/tests/queries.rs

```diff
@@ -10,7 +11,10 @@ use std::{fs, sync::Arc};
 
 #[test]
 fn queries() {
-    insta::glob!("data/*.json", |path| {
+    glob!("data/*.json", |path| {
+        let test_name = path.file_name().unwrap().to_str().unwrap();
+        println!("RUNNING: {test_name}");
```

Comment:
This has been on my mind for a bit already, we should write either a proc macro or a `build.rs` script to programmatically generate a separate test per each json file. I'll add a task in Linear.